src/Files.App/ViewModels/SettingsViewModels/FoldersViewModel.cs

```diff
@@ -250,6 +250,35 @@ public bool OpenFoldersNewTab
 			}
 		}
 
+		public bool SortInDescendingOrder
+		{
+			get => UserSettingsService.LayoutSettingsService.DefaultDirectorySortDirection == SortDirection.Descending;
+			set
+			{
+				if (value != (UserSettingsService.LayoutSettingsService.DefaultDirectorySortDirection == SortDirection.Descending))
```

Comment:
Can this be moved from `LayoutSettingsService` to `FoldersSettingsService`?

Reply:
Moved.